test/distributed/test_c10d_pypg.py

```diff
@@ -142,17 +158,31 @@ def test_ddp_with_pypg_with_grad_views(self):
         )
 
 
-class TestDDPWithWorkSubclass(AbstractDDPSingleRank, MultiProcessTestCase):
+class TestDDPWithWorkSubclass(AbstractDDPSingleRank, MultiThreadedTestCase):
     @property
     def use_wrapper(self):
         return False
 
 
-class TestDDPWithWorkWrapper(AbstractDDPSingleRank, MultiProcessTestCase):
+class TestDDPWithWorkWrapper(AbstractDDPSingleRank, MultiThreadedTestCase):
     @property
     def use_wrapper(self):
         return True
 
 
+class TestPyProcessGroup(TestCase):
```

Comment:
Directly inheriting from `unittest.TestCase` seems to cause CI errors.

Reply:
fixed